### ISSUE TITLE
Decimal / integer validation errors when no limits are provided

### ISSUE THREAD

**Issue by andybell** [MarineGEO/MarineGEO-template-builder#1]
Example when fieldType is "decimal" without any limit:

```
MarinegeoTemplateBuilder.classes.Field(sheet="Transect",fieldName="visibility", fieldType="decimal")
```

 ```
Traceback (most recent call last):
  File "/Users/ambell/Documents/MarineGEO-template-workbooks/reef-life-survey/rls-data-template.py", line 10, in <module>
    "Reef Life Survey", 'DEFAULT', protect=True, metadataValues=metadataValues)
  File "/Users/ambell/anaconda3/envs/MarineGEO-template-workbooks/lib/python3.6/site-packages/MarinegeoTemplateBuilder/construct.py", line 362, in main
    tabs = addSheets(workbook, listOfAllfields=attrib, tabsDict=tabs)
  File "/Users/ambell/anaconda3/envs/MarineGEO-template-workbooks/lib/python3.6/site-packages/MarinegeoTemplateBuilder/construct.py", line 72, in addSheets
    tabsDict[s] = addTabHeaderOnly(workbook, s, listoffields=sub)
  File "/Users/ambell/anaconda3/envs/MarineGEO-template-workbooks/lib/python3.6/site-packages/MarinegeoTemplateBuilder/construct.py", line 31, in addTabHeaderOnly
    createValidation(tab, col, s)  # creates validation for the column
  File "/Users/ambell/anaconda3/envs/MarineGEO-template-workbooks/lib/python3.6/site-packages/MarinegeoTemplateBuilder/validators.py", line 192, in createValidation
    validator = selectValidator(field)
  File "/Users/ambell/anaconda3/envs/MarineGEO-template-workbooks/lib/python3.6/site-packages/MarinegeoTemplateBuilder/validators.py", line 140, in selectValidator
    elif isfloat(field.minValue) & isfloat(field.minValue) & (float(field.minValue) < float(field.maxValue)):
TypeError: float() argument must be a string or a number, not 'NoneType'
```

**Comment by andybell** [comment]
Still not working correctly

`/Users/ambell/anaconda3/envs/MarineGEO-template-workbooks/lib/python3.6/site-packages/xlsxwriter/worksheet.py:1849: UserWarning: Parameter 'criteria' is required in data_validation()
  warn("Parameter 'criteria' is required in data_validation()")`

**Comment by andybell** [comment]
fixed sometime before 85c2581
